.bcr/metadata.template.json

```diff
@@ -2,9 +2,9 @@
   "homepage": "https://docs.aspect.build/rules/aspect_rules_webpack",
   "maintainers": [
     {
-      "name": "Greg Magolan",
-      "email": "greg@aspect.dev",
-      "github": "gregmagolan"
+      "email": "jason@aspect.build",
```

Comment:
I'm not sure if we want to keep Greg on here, and/or also add Alex? I'm not sure, but it might be good if I'm at least on the list so we don't have to bug busy people all the time...